### DISCUSSION TITLE
can i use yolov7 weights?

### DISCUSSION THREAD

**Discussion by BoscoShin** [mikel-brostrom/yolo_tracking#699] [Q&A]
i already trained the model(yolov7). so i have a model weights(yolov7 - best.pt) 

can i use yolov7 weights in STRONG SORT?

**Comment by mikel-brostrom** [comment]
Check `Custom object detection architecture` under README

**Reply by BoscoShin** [reply]
thank for reply.

i read a Custom object detection architecture under README.

i have some question.

do i pass the detector in this line? 

![image](https://user-images.githubusercontent.com/75238411/213595642-2b356316-8c43-4dc3-8ac9-cdc8e41bd465.png)

**Reply by mikel-brostrom** [reply]
Yolov5 and v7 are different architectures, so no, what you are trying to do is not possible at the moment

**Reply by BoscoShin** [reply]
ok i understand! 

thx
